[docs] CHANGELOG for v5.0.0-alpha.15 (#23383)
mui-org/material-ui: CHANGELOG.md

### [Versions](https://material-ui.com/versions/)

## 5.0.0-alpha.15

###### _Nov 4 2020_

Big thanks to the 20 contributors who made this release possible. Here are some highlights ✨:

- ⚛️ Add support for React 17 (#23311) @eps1lon.
  React 17 release is unusual because it doesn't add any new developer-facing features. It was released a couple of days ago. You can learn more about it in the [introduction post](https://reactjs.org/blog/2020/10/20/react-v17.html).
- 🛠 Introduce a new `@material-ui/unstyled` package (#23270) @mnajdova.
  The package will host the unstyled version of all the components. In this first iteration, only the Slider is available. You can find the component documented under the [same page](https://next.material-ui.com/components/slider-styled/#unstyled-slider) as the styled version.

  **Why an unstyled package?**

>   While many teams are successfully building custom design systems wrapping Material-UI, we [often hear](https://github.com/mui-org/material-ui/issues/6218) that Material Design or our styling solution are something they don't need. Some teams prefer SASS, others prefer to customize the components starting from a pristine state. What all these teams have in common is that they value the features coming from the components, like the accessibility logic.

Reviewer: ```suggestion
  While many teams are successfully building custom design systems wrapping Material-UI, we [occasionally hear](https://github.com/mui-org/material-ui/issues/6218) that Material Design or our styling solution are something they don't need. Some teams prefer SASS, others prefer to customize the components starting from a pristine state. What all these teams have in common is that they value the features coming from the components, such as accessibility.
```

"Some teams prefer SASS, others prefer to customize the components starting from a pristine state."

This reads like these are two alternative choices.

>   So the unstyled package goes one step down in the abstraction layer, providing more flexibility. Angular has introduced this approach two years ago. Their unstyled components account today for [25% of the usage](https://npm-stat.com/charts.html?package=%40angular%2Fmaterial&package=%40angular%2Fcdk&from=2017-11-03&to=2020-11-03).

Reviewer: ```suggestion
  The unstyled package goes one step down in the abstraction layer, providing more flexibility. Angular Material introduced this approach two years ago. Today their unstyled components account for [25% of the usage](https://npm-stat.com/charts.html?package=%40angular%2Fmaterial&package=%40angular%2Fcdk&from=2017-11-03&to=2020-11-03).
```

>   The second reason for the introduction of this package is to deliver on the [second theme](https://github.com/mui-org/material-ui/issues/22485) planned (not Material Design based).

Reviewer: ```suggestion
  The second reason for the introduction of this package is to lay the foundations for a planned [second theme](https://github.com/mui-org/material-ui/issues/22485) (not Material Design based).
```

  A note on the terminology. "unstyled" means that the components have the same API as the "styled" components but come without CSS. Material-UI also contains "headless" components that exposes a hook API, e.g. [useAutocomplete](https://next.material-ui.com/components/autocomplete/#useautocomplete) or [usePagination](https://next.material-ui.com/components/pagination/#usepagination).

>   This change is part of our strategy to iterate on v5 architecture with first the `Slider`. In the next release, we plan to replace the v4 slider with the v5 slider. Once the new approach is stress-tested and validated, we will roll out the new approach to all the components.

Reviewer: ```suggestion
  This change is part of our strategy to iterate on the v5 architecture with the `Slider` first. In the next release, we plan to replace the v4 slider with the v5 slider. Once the new approach is stress-tested and validated, we will roll it out to all the components.
```

### [Versions](https://material-ui.com/versions/)

## 5.0.0-alpha.15

###### _Nov 4 2020_

Big thanks to the 20 contributors who made this release possible. Here are some highlights ✨:

- ⚛️ Add support for React 17 (#23311) @eps1lon.

>   React 17 release is unusual because it doesn't add any new developer-facing features. It was released a couple of days ago. You can learn more about it in the [introduction post](https://reactjs.org/blog/2020/10/20/react-v17.html).

Author: ```suggestion
  React 17 release is unusual because it doesn't add any new developer-facing features. It was released a couple of days ago. You can learn more about it in the [introduction post](https://reactjs.org/blog/2020/10/20/react-v17.html). Material-UI now supports `^16.8.0 || ^17.0.0`.
```
